systems/controllers/CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if (eigen3_FOUND)
 
   pods_find_pkg_config(terrain-map)
 
+  pods_find_pkg_config(lcm)
```

Comment:
i think this should already be set upstream (it's in the main drake/CMakeLists.txt, and in a more robust form)